pandas/core/series.py

```diff
@@ -1096,8 +1096,7 @@ def iteritems(self):
         """
         return zip(iter(self.index), iter(self))
 
-    if compat.PY3:  # pragma: no cover
-        items = iteritems
```

Comment:
I think @shoyer is right here,; I think this will work if you do

``items=compat.iteritems`` which already has the correct PY2 (and 3) behavior